tools-local/unix/cmake/get-version.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+# Gets the version of CMake at the specified path.
+
+usage()
+{
+    echo "Usage: $0 cmake ToolName --ToolPath <path>"
```

Comment:
I agree dashed lowercase is much more conventional, but I think we need to have a higher-level discussion about being conventional per-platform vs. being the same as powershell. (There are situations like BfS where having the same parameters for both are helpful.)